packages/grafana-data/package.json

```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "build": "grafana-toolkit package:build --scope=data",
     "bundle": "rollup -c rollup.config.ts",
-    "clean": "rimraf ./dist ./compiled",
+    "clean": "rimraf ./dist ./compiled tsconfig.tsbuildinfo",
```

Comment:
I've realised that package build only worked the first time, as TS build cache prevent build to rerun, making `compiled` folder missing

Reply:
As this was unrelated to the main purpose of this PR, I decided to get rid of it. If this is not an issue for regular development workflow, I prefer to leave this unchanged

Reply:
I've opened #47492 to revert this as it can be an issue for local development and was also an issue in a CI pipeline recently. I'll likely tackle a revamp of package building in the near future.